api4/team.go

```diff
@@ -103,6 +104,29 @@ func getTeamsForUser(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getTeamsUnreadForUser(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireUserId()
+	if c.Err != nil {
+		return
+	}
+
+	if c.Session.UserId != c.Params.UserId && !app.SessionHasPermissionTo(c.Session, model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	// optional team id to be excluded from the result
+	teamId := r.URL.Query().Get("id")
```

Comment:
Can we update the query parameter to be "exclude_team" instead of "id"? I think that will make it's purpose clearer, sorry I should have mentioned this in my previous review